docs/01-introduction.Rmd

```diff
@@ -341,16 +341,23 @@ If you are working on an R Markdown post, but do not want **blogdown** to compil
 
 ## Other themes
 
-In Hugo, themes\index{Themes} control the entire appearance and functionality of your site. So, if you care a lot about the appearance of your website, you will probably spend quite a bit of time in the beginning looking for a Hugo theme that you like from the collection listed at https://themes.gohugo.io. Please note that not all themes have been tested against **blogdown**. If you find a certain theme does not work well with **blogdown**, you may report to https://github.com/rstudio/blogdown/issues, and we will try to investigate the reason, but it can be time-consuming to learn and understand how a new theme works, so we recommend that you learn more about Hugo by yourself before asking, and we also encourage users to help each other there.
+In Hugo, themes\index{Themes} control the entire appearance and functionality of your site. A Hugo theme is made with the following elements:
 
-After you have found a satisfactory theme, you need to figure out its GitHub username and repository name,^[For most themes, you can find this by navigating to the theme of your choice from http://themes.gohugo.io and then clicking on `Homepage`.] then either install the theme via\index{blogdown::install\_theme()} `blogdown::install_theme()`, or just create a new site under another new directory and pass the GitHub repository name to the `theme` argument of `new_site()`. We recommend that you use the second approach, because Hugo themes could be very complicated and the usage of each theme can be very different and highly dependent on `config.toml`. If you install a theme using `install_theme()` instead of `new_site()` you’ll need to manually create the `config.toml` file in the root directory of your website to match the newly installed theme.^[In a workaround, if you used `install_theme()` and set the `theme_example` argument to TRUE, then you can access an example `config.toml` file. In the `themes/` directory, navigate to the file for your newly downloaded theme and find `exampleSite/config.toml`. This file can be copied to your root directory (to replace the `config.toml` file from your original theme) or used as a template to correctly write a new `config.toml` file for your new theme.]
+1. Configuration files. This may be a single `config.yaml` or `config.toml` file, or some themes use a configuration directory also located in `config/_default/`. Read more about these files in Chapter \@ref(configuration).
```

Comment:
Curious: are themes with config directory working with **blogdown** ? 
I am not sure this is yet supported: https://github.com/rstudio/blogdown/issues/611

If it is not working yet, maybe we should warn to not choose a them with a config dir 🤷‍♂️

Reply:
Wowchemy has this structure, and the config.toml gets copied to the root. But other config options remain in the directory, see: https://alison.rbind.io/post/new-year-new-blogdown/#configure-your-site

Reply:
Thanks for the clarification.

Reply:
Let me test with `install_theme()` though, I could encourage use of the `update_config=FALSE` arg there.